Test/HaskellSupportSpec.hs

```diff
@@ -0,0 +1,18 @@
+{-|
+Module: Test.HaskellSupportSpec
+Copyright: (c) digitally induced GmbH, 2020
+-}
+module Test.HaskellSupportSpec where
+
+import Test.Hspec
+import IHP.Prelude
+import IHP.HaskellSupport
+
+tests = do
+    describe "HaskellSupport" do
+        describe "stripTags" do
+            it "should deal with empty input" do
+                stripTags "" `shouldBe` ""
+
+            it "should strip html tags and return the plain text" do 
+                stripTags "This is <b>Bold</b>" `shouldBe` "This is Bold"
```

Comment:
Is this functionality only meant as a utility, or is it used as a actual sanitizer?

Reply:
This function is used as a utility in the IHP email system. We use it to generate a plaintext email from the html email by removing all the html tags. [Like `text mail = stripTags (cs $ Blaze.renderHtml (html mail))`](https://github.com/digitallyinduced/ihp/blob/master/IHP/Mail/Types.hs#L45).